### PR TITLE
ADD: Add Sodar reader.

### PR DESCRIPTION
**Pull request by zssherman**
Adding a sodar reader for the atmos site. Reader reads in .mnd files. Will add tests and an example to this PR as well.

- [x] Tests added
- [x] Documentation reflects changes
- [x] PEP8 Standards or use of linter
- [x] Xarray Dataset or DataArray variable naming follows 'ds' or 'da' naming

### CODE REVIEW

File: act/io/sodar.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import fsspec
+import numpy as np
+import pandas as pd
+
+from act.io.noaapsl import filter_list
+
+
+def read_sodar(filepath):
+    """
+    Returns `xarray.Dataset` with stored data and metadata from a user-defined
```
Comment (mgrover1) [inline act/io/sodar.py:18]: Can you specify what type of sodar is used here? With a link to the instrument docs?

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small change - otherwise, this looks great!! Thanks for your hard work here @zssherman

File: act/io/sodar.py (hunk 1 above)
Comment (zssherman) [reply act/io/sodar.py:18]: Done

**Review by mgrover1 — APPROVED** [review]
Great! Thank you @zssherman

**Pull request merged**